Sources/Widgets/Widgets3D/InteractiveOrientationWidget/helpers.js

```diff
@@ -0,0 +1,101 @@
+import * as vtkMath from '@kitware/vtk.js/Common/Core/Math';
+import vtkOrientationMarkerWidget from '@kitware/vtk.js/Interaction/Widgets/OrientationMarkerWidget';
+import vtkAxesActor from '@kitware/vtk.js/Rendering/Core/AxesActor';
+import vtkInteractiveOrientationWidget from '@kitware/vtk.js/Widgets/Widgets3D/InteractiveOrientationWidget';
+
+export function majorAxis(vec3, idxA, idxB) {
+  const axis = [0, 0, 0];
+  const idx = Math.abs(vec3[idxA]) > Math.abs(vec3[idxB]) ? idxA : idxB;
+  const value = vec3[idx] > 0 ? 1 : -1;
+  axis[idx] = value;
+  return axis;
+}
+
+export function createOrientationMarkerWidget(interactor, mainRenderer) {
+  const axes = vtkAxesActor.newInstance();
+
+  const orientationWidget = vtkOrientationMarkerWidget.newInstance({
+    actor: axes,
+    interactor,
+    spawnInteractiveRenderer: true,
+  });
+  orientationWidget.setEnabled(true);
+  orientationWidget.setViewportCorner(
+    vtkOrientationMarkerWidget.Corners.BOTTOM_LEFT
+  );
+  orientationWidget.setViewportSize(0.1);
+  orientationWidget.setMinPixelSize(100);
+  orientationWidget.setMaxPixelSize(300);
+
+  orientationWidget.setParentRenderer(mainRenderer);
```

Comment:
NIP: pass viewportSize, minPixelSize, maxPixelSize and parentRenderer in `newInstance()`